css-pseudo-classes/styles.css

```diff
@@ -0,0 +1,20 @@
+button {
+  padding: 5px 10px;
+  width: 150px;
+  font-size: 20px;
+}
+
+button:hover {
+  color: white;
+  background-color: black;
+  cursor: pointer;
+}
+
+table.header thead > tr {
+  color: white;
+  background-color: black;
+}
+
+table.header tbody > tr:nth-child(odd) {
```

Comment:
This looks **_VERY_** similar to what I did in my code, and my selectors were needlessly complicated on purpose for code reading practice. This works so I won't ask you to change it, but please be careful not to copy the code I present in Q&A to complete your exercises. "Completion" is _far_ less important than taking the time to do it yourself, so that you can more fully _understand_ the concept.

Reply:
Yes sir. In the beginning, I wrote it differently by using table and h1 selectors in the styles.css directory to manipulate. I also used tr selector to manipulate. 

Upon seeing your code on screen, I understood targeting the child element of the thead and tbody elements. So I wrote it afterwards. And got it. Completion second, understanding first.